typings/index.d.ts

```diff
@@ -872,7 +872,9 @@ export class Guild extends AnonymousGuild {
   public edit(data: GuildEditData, reason?: string): Promise<Guild>;
   public editWelcomeScreen(data: WelcomeScreenEditData): Promise<WelcomeScreen>;
   public equals(guild: Guild): boolean;
-  public fetchAuditLogs(options?: GuildAuditLogsFetchOptions): Promise<GuildAuditLogs>;
+  public fetchAuditLogs<T extends keyof GuildAuditLogsIds | GuildAuditLogsAction = 'ALL'>(
```

Comment:
Consider making a type alias for `keyof GuildAuditLogsIds | GuildAuditLogsAction`:

```ts
export type AuditLogsResolvable = keyof GuildAuditLogsIds | GuildAuditLogsAction;
```

Reply:
Done in 626aab1!